helm-chart/splunk-operator/crds/enterprise.splunk.com_licensemasters.yaml

```diff
@@ -3779,29 +3884,3 @@ spec:
     storage: true
     subresources:
       status: {}
-  - name: v1
-    schema:
-      openAPIV3Schema:
-        properties:
-          apiVersion:
-            type: string
-        type: object
-        x-kubernetes-preserve-unknown-fields: true
-    served: true
-    storage: false
-  - name: v2
-    schema:
-      openAPIV3Schema:
-        properties:
-          apiVersion:
-            type: string
-        type: object
-        x-kubernetes-preserve-unknown-fields: true
-    served: true
-    storage: false
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: null
-  storedVersions: null
```

Comment:
also is this also expected?

Reply:
added back v1 and v2 to license master and cluster master